pulsar-client-tools-test/src/test/java/org/apache/pulsar/client/cli/PulsarClientToolTest.java

```diff
@@ -207,7 +186,7 @@ public void testDurableSubscribe() throws Exception {
         Assert.assertFalse(future.isCompletedExceptionally());
         future.get();
         //wait for close
-        Thread.sleep(2000);
+        Awaitility.waitAtMost(Duration.ofDays(2000));
```

Comment:
Was this change intentional?

Reply:
I'm so sorry, This is my mistake, I will fix it ASAP.

Reply:
#11176 FIxed this mistake